docs/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -0,0 +1,125 @@
+.. _how_to_guides__creating_batches__how_to_load_a_database_table_view_or_a_query_result_as_a_batch:
+
+How to load a database table, view, or query result as a batch
+==============================================================
+
+This guide shows how to get a :ref:`batch <Batch>` of data that Great Expectations can validate from a SQL database. When you validate data using SQL, all compute is managed by your database, and only Validation Results are returned.
+
+
+.. admonition:: Prerequisites -- this how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <getting_started>`
+  - :ref:`Configured a SQL datasource <_how_to_guides__configuring_datasources>`
+  - Identified a ``table`` name, ``view`` name, or ``query`` that you would like to use as the data to validate.
+  - Optionally, configured a Batch Kwargs Generator to support inspecting your database and dynamically building Batch Kwargs.
```

Comment:
After the batch refactor, using a configured Generator will be the preferred method, correct?

Reply:
Yes. It will still be optional at that point technically, however, I think.

Reply:
+1